### ISSUE TITLE
Proof points related to financial  allocations that support digital accessibility maturity missing

### ISSUE THREAD

**Issue by jeffkline** [w3c/maturity-model#244]
There are currently no proof points related to fiscal aspects of implementing the maturity model. Without budget for the initiative, probability of success (full maturity ) is low.

I propose adding a  proof point to each of the dimensions for ensuring that budget is planned and committed  to bring the dimension proof points to full maturity.

Sample language:

Financial planning in place and funding committed for advancing this dimension's proof points to full maturity (optimize stage)
